fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
So now I'm confused. You told me to remove the # define _INC_TIME_INL, but if I create a time() function I'll just have the same problem.